src/utils/restrictToSameSchool.js

```diff
@@ -0,0 +1,18 @@
+const { Forbidden, BadRequest } = require('../errors');
+const { userRepo } = require('../components/user/repo/index');
+const { equal: equalIds } = require('../helper/compare').ObjectId;
+
+const restrictToSameSchool = async (id, account, app) => {
```

Comment:
id -> userId
The same app should not pass deeper if it is not really needed and no other way to handle it.